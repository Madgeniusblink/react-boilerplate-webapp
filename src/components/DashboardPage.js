import React from 'react'

const DashboardPage = () => (
    <div>
        <div>
            MadGeniusBlink dashboard in creation...
        </div>
    </div>
)

export { DashboardPage as default }