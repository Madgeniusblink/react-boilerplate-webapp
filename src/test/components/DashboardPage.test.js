// react-test-renderer
import React from 'react'
import { shallow } from 'enzyme'
import DashboardPage from '../../components/DashboardPage'

test('should render Madgenius Dashboard Page', () => {
    const wrapper = shallow(<DashboardPage />)
    expect(wrapper).toMatchSnapshot()
})
