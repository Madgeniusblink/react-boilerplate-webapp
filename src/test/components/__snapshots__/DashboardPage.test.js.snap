// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Madgenius Dashboard Page 1`] = `
<div>
  <div>
    MadGeniusBlink dashboard in creation...
  </div>
</div>
`;
